usr/share/rear/prep/USB/default/480_initialize_write_protect_settings.sh

```diff
@@ -0,0 +1,4 @@
+# USB output typically resides on a writable disk device, which should be protected against
+# accidental overwriting by rear recover. This code registers it as write protected.
+
+WRITE_PROTECTED_PARTITION_TABLE_UUIDS+=( $(lsblk --output PTUUID --noheadings --nodeps "$USB_DEVICE") )
```

Comment:
older lsblk do not support PTUUID
so some fallback code if partition_table_uuid is empty should be there.

Reply:
Same here: There are intentionally no quotes around `$(...)`, so nothing gets added to the array.